src/helpers/helpersAPI.js

```diff
@@ -17,4 +17,36 @@ const sortLocationsByDistance = (locations = []) => {
   });
 };
 
-export { getLocationsFromResponseAPI, sortLocationsByDistance };
+const MEASUREMENTS = {
+  METER: 'mts.',
+  KILOMETERS: 'kms.',
+  MILES: 'miles.',
+};
+
+const convertDistanceMts = (distanceInMts = 0, measurement = 'miles') => {
+  let result = distanceInMts;
+
+  switch (measurement) {
+    case MEASUREMENTS.METERS:
+      result = distanceInMts;
+      break;
+    case MEASUREMENTS.KILOMETERS:
+      result = distanceInMts / 1000;
+      break;
+    case MEASUREMENTS.MILES:
+      result = distanceInMts * 0.000621371;
+      break;
+    default:
+      result = distanceInMts;
+      break;
+  }
```

Comment:
This is good forward thinking! We could have the measurement in the dialog change depending on the users location! (Not for this ticket though)

Reply:
Yes! I just create the ticket [TCL30-62](https://the-collab-lab.atlassian.net/browse/TCL30-62) for it ^.